frontend/src/lib/components/DefaultSelector.svelte

```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
+	import { ChevronDown, ChevronUp, Check } from 'lucide-svelte';
+
+	export let options: string[] = [];
```

Comment:
Please use svelte 5 features (runes) for props, states and slots

$props instead of export
$state for state
snippets for slots